src/game/game/game.c

```diff
@@ -70,8 +70,9 @@ static int32_t Game_Control(int32_t nframes, GAMEFLOW_LEVEL_TYPE level_type)
             }
         }
 
-        if ((g_InputDB.option || g_Input.save || g_Input.load
-             || g_OverlayFlag <= 0)
+        if ((g_Config.enable_buffering ? g_Input.option
+                                       : g_InputDB.option || g_Input.save
```

Comment:
Let's wrap the ternary operator with `()`, since now it's not evident if it's meant to say `(a ? b : c) || d || e` or `a ? b : (c || d || e)`.